tests/autoyast/verify_lvm_partitions.pm

```diff
@@ -0,0 +1,28 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+#
+# Summary: Verify lvm partitions after autoyast installation
+# Maintainer: qa-sle-yast <qa-sle-yast@suse.com>
+
+use strict;
+use warnings;
+use parent 'installbasetest';
+use testapi;
+use scheduler 'get_test_suite_data';
+
+sub run {
+    my $test_data = get_test_suite_data();
+    my $vg        = $test_data->{lvm}->{vg};
+    assert_script_run("vgdisplay $vg");
+    foreach my $lv (@{$test_data->{lvm}->{lvs}}) {
+        assert_script_run("lvdisplay /dev/$vg/$lv");
```

Comment:
I'd recommend to add some validation on the output of this commands.

Reply:
I guess command will return non-zero code in case we specify non-existing device. @JRivrain could you please confirm this?

Reply:
Yes, that is the idea here, assert_script_run will fail if device does not exist.

Reply:
So @jknphy is it still an objection for you, considering my response ? If not, I'd consider removing WIP at that point.

Reply:
I guess we are fine to proceed.